root_draft_metadata_ecr.tf

```diff
@@ -0,0 +1,26 @@
+module "draft_metadata_validator_lambda_ecr" {
+  source          = "./da-terraform-modules/lambda"
+  function_name   = "tdr-draft-metadata-validator-v2-${local.environment}"
+  handler         = "uk.gov.nationalarchives.draftmetadatavalidator.Lambda::handleRequest"
+  tags            = local.common_tags
+  use_image       = true
+  image_url       = "${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/draft-metadata-validator:v41"
```

Comment:
The `v41` should not be hard coded. Should use the environment tag, ie: `... draft-metadata-validator:${local.environment}`

Reply:
I have changed it. Had hard coded the value to simplify release.
I am creating another github workflow to allow environment tagging to make it simpler